### ISSUE TITLE
Don't send `StreamSpecification: null`

### ISSUE THREAD

**Issue by dimaqq** [HENNGE/aiodynamo#11]
https://github.com/HENNGE/aiodynamo/blob/81b6f98111ce4bed41aeb8d07747695af836c76a/src/aiodynamo/client.py#L41-L49

Sends `StreamSpecification=None` when none is supplied by the caller.
Perhaps `aiodynamo` should not inject this key if the value `is None`.

It seems that's OK for AWS SaaS dynamo, dynamodb-local and dynalite, but not ScyllaDB:

https://github.com/scylladb/scylla/issues/5796

**Comment by dimaqq** [comment]
It could also be that I'm wrong and scylla doesn't grok whatever `boto*` sends in the end...

**Comment by nyh** [comment]
I'm having trouble reproducing this in boto3 in Python, trying to pass StreamSpecification=None doesn't pass boto3's internal checks:
```
E               ParamValidationError: Parameter validation failed:
E               Invalid type for parameter StreamSpecification, value: None, type: <type 'NoneType'>, valid types: <type 'dict'>
```
If I try to pass StreamSpecification={}, DynamoDB doesn't allow this either.
Anyway, I'll prepare a patch for you to test, and test what I can in the boto3-based tests.

**Comment by dimaqq** [comment]
I got a notification on the comment here, but the comment is gone.
A clarification on what to pass for `StreamSpecification`: it needs to be a default "object" that boto accepts, of course, sorry the library I used was too clever:
https://github.com/HENNGE/aiodynamo/blob/323727d5367fca4865486b7dcb9cffdaf1948cd2/src/aiodynamo/models.py#L106-L114

**Comment by nyh** [comment]
Sorry, I accidentally posted a response here instead of https://github.com/scylladb/scylla/issues/5796, and then deleted it when I realized my mistake.

In any case, I already committed to Scylla a patch to be more forgiving about the StreamSpecification parameter. If this parameter is a null, or if it's a real map with the "Enabled=false" parameter, it is ignored instead of generating an error. So hopefully your code will work, unchanged, on Scylla.
